Makefile

```diff
@@ -62,3 +70,15 @@ generate-code:
 patch: generate-code
 	awk '($$1 != "\"github.com/gorilla/mux\"") {print}' auto-generated/api/go/api_default.go > ${TMP_FILE}
 	mv ${TMP_FILE} auto-generated/api/go/api_default.go
+
+.PHONY: build
+build: compile
+	docker build . -t ${IMG}; cd ..
+
+.PHONY: docker-push
+docker-push:
```

Comment:
can you change it according to https://github.com/fybrik/fybrik/blob/d97245e50c1d0384046a9b601d0166c821617f38/hack/make-rules/docker.mk#L11